config/v1/types_authentication.go

```diff
@@ -14,8 +14,7 @@ type Authentication struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// spec holds user settable values for configuration
-	// +kubebuilder:validation:Required
-	// +required
+	// +optional
```

Comment:
I'm specifying spec, don't I need this on teh type field?

Reply:
Nah this works here. There was some bug with that around named types but I just confirmed manually